doc/howto/edit_your_dockerfile.md

```diff
@@ -0,0 +1,49 @@
+# How to edit your app's Dockerfile
+
+To set up custom programs for your app, such as third-party tools, you can edit your Dockerfile, which contains the setup code needed to build up your container.
+
+A helpful resource is the [Dockerfile Best Practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices) guide.
+
+The base KBase Docker image contains a KBase Ubuntu image, but not much else. You will need to include any dependencies, including the installation of whatever tool you are wrapping, to build a custom Docker image that can run your Module.
```

Comment:
Actually the KBase Docker image also contains the dependencies necessary to run a JSON-RPC server in the supported language, as well as a core set of KBase API clients